specs/shell/array_append_values.yaml

```diff
@@ -0,0 +1,16 @@
+---
+name: Append value(s) to an array
+command: "{{array_name}}+=({{values}})"
+tags:
+  - shell
+description: Loops through an array, running a command on each value.
+arguments:
+  - name: array_name
+    description: The name of the array to loop through
+    default_value: ~
+  - name: values
+    description: The values to append to the array, space-separated
+    default_value: ~
+source_url: "https://opensource.com/article/18/5/you-dont-know-bash-intro-bash-arrays"
+author: Wyatt-Stanke
```

Comment:
Since these workflows use shell-specific syntax, we need to set the `shell` field so they don't appear for all shells. Specifically:

```yaml
shells:
  - Zsh
  - Bash
```

Reply:
Fixed!